### PR TITLE
Create Tether.md

### PR DESCRIPTION
**Pull request by Desser**

### CODE REVIEW

**Comment by Desser** [comment]
Ready

**Comment by Desser** [comment]
Added

**Comment by Desser** [comment]
Correction

File: people.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+1. [Satoshi Nakamoto](/people/SatoshiNakamoto.md)
+2. [Vitalik Buterin](/people/VitalikButerin.md)
+3. [Gavin Wood](/people/GavinWood.md)
+4. [Dan Larrimer](/people/DanLarrimer.md)
+5. [Alexander Ivanov](/people/AlexanderIvanov.md)
+6. [Patrick Dai](/people/PatrickDai.md) 
+7. [Zooko Wilcox](/people/ZookoWilcox.md)
+8. [Reggie Middleton](/people/ReggieMiddleton.md) 
+9. [Brock Pierce](/people/BrockPierce.md)
+10.[Reeve Collins](/people/ReeveCollins.md)
```
Comment (akhavr) [inline people.md]: Here you break the numbered list.  Please, eyecheck your changes in github before submitting.

File: README.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ## Welcome to KeyCryptoVC Base
   		  
  - [Projects](projects/projects.md)
```
Comment (akhavr) [inline README.md]: Here you link to `projects/projects.md` but you keep putting changes into `project.md`

Comment (akhavr) [inline README.md]: Just go and click through https://github.com/Desser/base/tree/patch-10 ?

**Comment by Desser** [comment]
Correction

**Comment by akhavr** [comment]
Why https://github.com/Desser/base/blob/patch-10/projects/projects.md has spots 3-9 empty?

If that's ordering by market cap, let don't do it.  Instead sort alphabetically

**Comment by akhavr** [comment]
Good example of alphabet grouping is https://en.wikipedia.org/wiki/Category:Computational_science

Don't do exactly this  right now, but hopefully you get the point.

File: projects/projects.md [hunk 3]
```diff
@@ -1,2 +1,22 @@
...
 1.  [Bitcoin](/projects/bitcoin.md)
 2.  [Ethereum](/projects/ethereum.md)
+3.  
+4.  
+5.  
+6.  
+7.  
+8.  
+9.  
+10. [Stratis](/projects/Stratis.md)
```
Comment (akhavr) [inline projects/projects.md]: This PR adds only Tether.  Why links to other proejcts?

**Pull request closed without merging**